src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -274,31 +274,31 @@
     <value>Attributes can define mandatory arguments that must be specified when you apply the attribute to a target. These are also known as positional arguments because they are supplied to attribute constructors as positional parameters. For every mandatory argument, the attribute should also provide a corresponding read-only property so that the value of the argument can be retrieved at execution time. Attributes can also define optional arguments, which are also known as named arguments. These arguments are supplied to attribute constructors by name and should have a corresponding read/write property.</value>
   </data>
   <data name="DefineAccessorsForAttributeArgumentsMessageDefault" xml:space="preserve">
-    <value>Add a public read-only property accessor for positional argument {0} of Attribute {1}</value>
+    <value>Add a public read-only property accessor for positional argument '{0}' of Attribute '{1}'</value>
   </data>
   <data name="DefineAccessorsForAttributeArgumentsMessageRemoveSetter" xml:space="preserve">
-    <value>Remove the property setter from {0} or reduce its accessibility because it corresponds to positional argument {1}</value>
+    <value>Remove the property setter from '{0}' or reduce its accessibility because it corresponds to positional argument '{1}'</value>
   </data>
   <data name="DefineAccessorsForAttributeArgumentsMessageIncreaseVisibility" xml:space="preserve">
-    <value>If {0} is the property accessor for positional argument {1}, make it public</value>
+    <value>If '{0}' is the property accessor for positional argument '{1}', make it public</value>
```

Comment:
What about `public`? Is the the keyword or a short-hand for public and protected?